awslabs/ec2-spot-workshops
# Update Running EC2 workloads at scale workshop

*Description of changes:*
* Updated the Spot Interruption section to use https://github.com/awslabs/ec2-spot-labs/tree/master/ec2-spot-interruption-handler as per #66 
* Updated the workshop to use *capacity-optimized* Spot Allocation Strategy
* Updated sed commands to use standard format " s/a/b/g" as per prior discussions. Left only ARN substitutions with "s#a#b#g" as otherwise it fails due to special characters
* Added a few clarifications to make attendees validate command execution
* Added a step to check savings
* Re-enabled Load Balancer stickiness as it's needed by Koel
* Adjusted ALB deregistration delay to 90 seconds as per latest guidance [here](https://aws.amazon.com/blogs/compute/running-web-applications-on-amazon-ec2-spot-instances/). 
* Update CodeDeploy install-dependencies.sh to include gcc-g++
* Use SSM Parameter Store to get the latest Amazon Linux 2 AMI

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
